environments/dev/dev.tfvars

```diff
@@ -0,0 +1,19 @@
+project                    = "hmcts"
```

Comment:
don't you need to change the azure pipelines file as well?

Reply:
would help, wouldn't it? :D added